### PR TITLE
Fix mau-extratests on public cloud

### PR DESCRIPTION
**Pull request by pdostal**
Hello,

here are some small fixes of several tests from the `mau-extratests-console` tests suite.

- Related ticket: [poo#54842](https://progress.opensuse.org/issues/54842)
- Needles: [os-autoinst-needles-sles#1234](https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/1234)
- Verification run: [latest](http://pdostal-server.suse.cz/tests/5296)

### CODE REVIEW

File: lib/services/ntpd.pm [hunk 1]
```diff
@@ -59,7 +59,7 @@ sub check_function {
     validate_script_output("date", sub { m/2018/ });
     systemctl("restart ntpd.service");
     sleep(120);
-    validate_script_output("date", sub { not m/2018/ });
+    validate_script_output("date", sub { not m/2018/ }, timeout => 300);
```
Comment (pdostal) [inline lib/services/ntpd.pm]: @KatkaSuSe is this OK?

File: lib/version_utils.pm [hunk 2]
```diff
@@ -422,6 +422,7 @@ sub is_server {
     return 1 if is_sles4sap();
     return 1 if is_sles4migration();
     return 1 if get_var('FLAVOR', '') =~ /^Server/;
+    return 1 if get_var('PUBLIC_CLOUD');
```
Comment (pdostal) [inline lib/version_utils.pm]: @mimi1vx IMO Public Clouod should be always server.

File: tests/console/mta.pm [hunk 3]
```diff
@@ -22,11 +22,17 @@ sub run {
...
-    systemctl 'is-enabled postfix';
-    systemctl 'is-active postfix';
-    systemctl 'status postfix';
+    unless (get_var('PUBLIC_CLOUD')) {
+        # check if postfix is installed, enabled and running
+        assert_script_run 'rpm -q postfix';
+        systemctl 'is-enabled postfix';
+        systemctl 'is-active postfix';
+        systemctl 'status postfix';
+    } else {
```
Comment (pdostal) [inline tests/console/mta.pm]: I discussed this on `#public-cloud-users`

File: tests/console/mutt.pm [hunk 4]
```diff
@@ -31,7 +31,7 @@ sub run {
     my $self = shift;
     $self->select_serial_terminal;
 
-    zypper_call("in mutt", exitcode => [0, 102, 103]) if (is_tumbleweed || is_jeos);
+    zypper_call("in mutt", exitcode => [0, 102, 103]) if (is_tumbleweed || is_jeos || get_var('PUBLIC_CLOUD'));
```
Comment (pdostal) [inline tests/console/mutt.pm]: @baierjan I believe `mutt` is missing on purpose on our Public Cloud instances. Do you agree?

File: tests/console/suse_module_tools.pm [hunk 5]
```diff
@@ -38,7 +38,7 @@ sub run {
     # testing modsign-verify command
     if (is_sle('<15-sp1')) {
         # Run command and grep correct output
-        assert_script_run("modsign-verify $pth | grep 'good signature\\|bad signature\\|certificate not found\\|module not signed\\|other error'");
+        assert_script_run("modsign-verify $pth | grep 'good signature\\|bad signature\\|certificate not found\\|module not signed\\|other error\|Invalid signature format'");
```
Comment (pdostal) [inline tests/console/suse_module_tools.pm]: @BillAnastasiadis this is your suggested solution. Is that all right?

Comment (BillAnastasiadis) [reply tests/console/suse_module_tools.pm]: Yes, that seems correct. Should fix the problem.

Comment (BillAnastasiadis) [reply tests/console/suse_module_tools.pm]: Just a comment - I suggest escaping the last \ (other error\\|Invalid signature format'")

File: tests/console/mutt.pm (hunk 4 above)
Comment (baierjan) [reply tests/console/mutt.pm]: Public Cloud and jeos have probably a lot in common. For example missing almost everything by default :smile:

**Pull request merged**